### PR TITLE
Segd2ph5 rem sys exit logging format

### PR DESCRIPTION
**Pull request by damhuonglan**
This PR work on segd2ph5.py on the following issues:
#215 remove call for sys.exit()
Apply LOGGING FORMAT

### Checklist
- [ ] This PR is not directly related to an existing issue (which has no PR yet).
- [ ] All tests pass.
- [ ] Any new or changed features have are documented.
- [ ] Changes have been added to `CHANGELOG.txt` .
- [ ] First time contributors have added your name to `CONTRIBUTORS.txt` .

### CODE REVIEW

File: ph5/utilities/segd2ph5.py [hunk 1]
```diff
@@ -16,9 +16,11 @@
 import re
 from math import modf
 from ph5.core import experiment, columns, segdreader
+from ph5 import LOGGING_FORMAT
```
Comment (dsentinel) [inline ph5/utilities/segd2ph5.py]: This shouldn't be necessary.
The logger in init already set the formatter

File: ph5/utilities/segd2ph5.py [hunk 2]
```diff
@@ -125,7 +127,7 @@ def fn_sort(a, b):
     try:
         fh = file(infile)
     except BaseException:
```
Comment (dsentinel) [inline ph5/utilities/segd2ph5.py]: `BaseException`'s should be `Exception`

File: ph5/utilities/segd2ph5.py [hunk 3]
```diff
@@ -219,21 +221,21 @@ def get_args():
...
-
-    logging.basicConfig(
-        filename=os.path.join('.', "segd2ph5.log"),
-        format="%(asctime)s %(message)s",
-        level=logging.INFO
-    )
+        raise Exception("No outfile (PH5) given.\n")
+
+    # Write log to file
+    ch = logging.FileHandler(os.path.join('.', "segd2ph5.log"))
```
Comment (dsentinel) [inline ph5/utilities/segd2ph5.py]: The logger will default to the current dir so:
```
logging.FileHandler("segd2ph5.log")
```

File: ph5/utilities/segd2ph5.py [hunk 4]
```diff
@@ -16,9 +16,11 @@
 import re
 from math import modf
 from ph5.core import experiment, columns, segdreader
+from ph5 import LOGGING_FORMAT
 from pyproj import Proj, transform
 
-PROG_VERSION = "2019.14"
+PROG_VERSION = "2019.252"
+LOGGER = logging.getLogger(__name__)
```
Comment (dsentinel) [inline ph5/utilities/segd2ph5.py]: Lower case `logger` instead of `LOGGER`

File: ph5/utilities/segd2ph5.py (hunk 1 above)
Comment (damhuonglan) [reply ph5/utilities/segd2ph5.py]: @dsentinel This one is necessary for setting format to write log to file (line 227-232). If I don't setFormatter for this, the format on screen's log isn't affected, but the log file's will be different.

Comment (dsentinel) [reply ph5/utilities/segd2ph5.py]: Nevermind, you are creating another handler for file logging.

**Comment by dsentinel** [comment]
Nice, thanks!

**Pull request merged**